docs/UserGuide.md

```diff
@@ -647,12 +646,13 @@ company. Use the `find` command to quickly locate the company you want!_
 **:information_source: Notes:**<br>
 * The `find` command searches only the company name.
 * Companies with names that contain **any** of the keywords will be returned.
-* The case of the keyword doesn't matter (e.g. `tiktok` will match `TikTok`).
+* Keyword is case-insensitive (e.g. `tiktok` will match `TikTok`).
 * The order of your keywords doesn't matter (e.g. `tiktok google` will match `google tiktok`).
+* LinkMeIn's `find` command will always find from the **full** list of companies. This means that if you have previously used `filter s/pa`, performing `find`, will not search from the filtered list of companies.
```

Comment:
Is there a need to specify an example? Perhaps just saying previously used `filter` might be better

Reply:
true ok removed the example